src/QueryBuilder/HasPriceBetweenQueryBuilder.php

```diff
@@ -91,6 +92,11 @@ private function resolveBasePrice(string $price): int
     private function convertFromString(string $price): int
     {
         $transformer = new SyliusMoneyTransformer(2, false, SyliusMoneyTransformer::ROUND_HALF_UP, 100);
-        return $transformer->reverseTransform($price);
+
+        try {
+            return $transformer->reverseTransform($price);
+        } catch (TransformationFailedException $exception) {
+            return 0;
```

Comment:
I think we should handle invalid input in validator. If we have validation for this, we shouldn't do it here.

Reply:
I removed try-catch here

Reply:
I've added code that replaces invalid values with empty strings(empty string is default value in form causes no filtering).
https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/pull/149/files#diff-97fe6302999817ef4724d7b9c6b6ee16c0044cf0fc1181b95cd8ded3b792fb4bR72
https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/pull/149/files#diff-97fe6302999817ef4724d7b9c6b6ee16c0044cf0fc1181b95cd8ded3b792fb4bR93